src/iron_math.py

```diff
@@ -1,16 +1,35 @@
 import math
+import arcade
 from typing import Tuple
 
 
-def rotate_vec2(point: Tuple[float, float], radians: float, /) -> float:
+def rotate_vec(point: Tuple[float, float], radians: float, /) -> float:
```

Comment:
Yeah, if "vector" is a term of art in game dev, we can leave it, but I find it really confusing b/c it's not the meaning of vector in either linear algebra nor C++, which are the contexts most familiar to me.

Reply:
Yeah, here are some references:

Unity3d, which also supports 2d games, has Vector3 and Vector2:
https://docs.unity3d.com/ScriptReference/Vector2-ctor.html
It describes an object's (relative) position with a `Transform`, which stores the Cartesian coordinates in a `Vector3`
https://docs.unity3d.com/ScriptReference/Transform.html

We're not yet using the arcade engine's PyMunk physics integration (we may never?) but I googled their terminology.  PyMunk is a wrapper around Chipmunk, so I looked at Chipmunk's docs:
http://chipmunk-physics.net/release/ChipmunkLatest-Docs/#Basics-Types

Reply:
Googling some more, it seems the term Vector may in fact be used *because* games use linear algebra to manipulate them.  I haven't gotten into the details much.

Reply:
I now concur with @cspotcode that "vector" is a term of art, which means a Cartesian position vector.

Reply:
Small correction: it does not mean solely Cartesian position vector.  It means an ordered sequence of numbers, which is used to represent many things: rotations, position, scale, velocity.